democracylab/emails.py

```diff
@@ -205,12 +220,23 @@ def send_volunteer_application_email(volunteer_relation, is_reminder=False):
         project=project.project_name,
         role=role_text)
     email_template = HtmlEmailTemplate()\
-        .header("You Have a New Volunteer!")\
+        .subheader("Opportunity Information:")\
+        .text_line("Title: {role}".format(role=role_details.display_name))\
+        .text_line("Organization: {project}".format(project=project.project_name))\
+        .text_line("Date: {currentdate}".format(currentdate=models.DateTimeField(auto_now_add=True, null=True)))\
```

Comment:
models.DateTimeField is specifically for declaring a database column.  If you want to get the current date/time, use timezone.now() and pass it into date_helpers.datetime_to_string.

Example: https://github.com/DemocracyLab/CivicTechExchange/blob/fa6633d16eadc0cdacea7c052b68fdf6b1fc8e98/democracylab/emails.py#L253

Reply:
Got it. I ... hm, I must not have committed a change I made where I copied from a better example in emails.py but it wasn't timezone.now() so I'll fix this, thanks.